scripts/entity/tactical/enemies/rf_skeleton_heavy_lesser.nut

```diff
@@ -1,3 +1,4 @@
+//Ancient Praetorian
```

Comment:
Using the very first line for that feels a bit odd.
Maybe it should go on the right side of the inherit line? Separated by a single tab or space
I would also add a single space bar after the slashes.

e.g.:
`this.rf_skeleton_heavy_lesser <- ::inherit("scripts/entity/tactical/skeleton", { // Ancient Praetorian`

Reply:
I'm neutral as to which is nicer. Will see what Midas thinks

Reply:
I prefer the comment on the first line, as it is much more visible the moment you open the file. But I agree with Darxo that there should be a space after the slashes so `// Ancient Praetorian`

Reply:
Alright, let's leave it in the first row with an additional space each

Reply:
Done